### ISSUE TITLE
Can't connect Spotify

### ISSUE THREAD

**Issue by VeldenD** [bitfocus/companion-module-spotify-remote#33]
Getting te message INVALID_CLIENT: Invalid redirect URI eversince

**Comment by CoBot3000** [comment]
Hey Ya, 

I also had the same problem. 
I managed to get it to work when I took all the extra line of text out: 
https://accounts.spotify.com/authorize?client_id=605ca0d1991a4661848b7065bd8ac718&response_type=code&redirect_uri=https://spotauth.github.io

So just make sure you have your client id in there.

But I still can't get it to connect.
I have been able to get the access token or refresh token.
I have the Approval code in there and it says to refresh which i have done and still nothing. 

Any thoughts.
